tests/tracing/test_opentelemetry.py

```diff
@@ -0,0 +1,70 @@
+from unittest.mock import ANY, Mock, call, patch
```

Comment:
Perfect use-case where ChatGPT shines! I provided the implementation and asked it to write a test for each test method; I just had to make some minor adjustments :1st_place_medal:  

I was hesitant to add these types of tests since the implementation is just a tiny wrapper that barely does anything, and the most significant bits are already tested in the consumer/producer. But at least the `extract_links` includes some logic, so I thought I might as well add some basic tests for each method. Also, we had full coverage before.